File: Modul 4/PRAK405_2410817310016_ARIFFATAHILLAHPERMANA.py
n, kelipatan = map(int, input("").split())
total = 0
for i in range(1, n + 1):
    jumlah_kelipatan = 0
    for j in range(1, i + 1):
        jumlah_kelipatan += j * kelipatan
        if j > 1:
            print(" + ", end="")
        print(f"({j} * {kelipatan})", end="")
    
    print(f" = {jumlah_kelipatan}")
    total += jumlah_kelipatan
print(total)

File: Modul 4/PRAK405_2410817310016_ARIFFATAHILLAHPERMANA.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    int total = 0; 
    scanf("%d %d", &n, &kelipatan);
    for (int i = 1; i <= n; i++) {
        int jumlah_kelipatan = 0; 
        for (int j = 1; j <= i; j++) {
            jumlah_kelipatan += j * kelipatan;
            if (j > 1) {
                printf(" + ");
            }
            printf("(%d * %d)", j, kelipatan); 
        }
        printf(" = %d\n", jumlah_kelipatan); 
        total += jumlah_kelipatan; 
    }
    printf("%d\n", total);

    return 0;
}